lib/index.js

```diff
@@ -212,7 +212,7 @@ class Ream extends Event {
         const html = await this.renderer.renderToString(context)
         const { start, end } = await renderTemplate(context)
         const targetPath = this.resolveOutDir(
-          `generated/${route.replace(/\/?$/, '/index.html')}`
+          `generated/${route.replace(/(\/?$|\/?\?.*$)/, '/index.html')}`
```

Comment:
shouldn't it rather be something like `/\/?(\?.*)?$/`

Reply:
Yes, I will fix it.